schemas/development/xarf_shared.schema.json

```diff
@@ -257,6 +257,45 @@
         }
       ]
     },
+    "AccountBasedReport": {
+      "allOf": [
+        {
+          "$ref": "#/properties/ReportBase"
+        },
+        {
+          "$ref": "#/properties/Date"
+        },
+        {
+          "type": "object",
+          "required": ["Account"],
+          "properties": {
+            "Account": {
+              "type": "object",
+              "minProperties": 1,
+              "description": "Account data of the reportee",
+              "properties": {
+                "AccountName": {
+                  "type": "string",
+                  "description": "reported account name",
+                  "not": {
+                    "description": "email based user accounts should go to the AccountEmail field",
+                    "format": "email"
+                  }
+                },
+                "AccountEmail": {
+                  "type": "string",
+                  "description": "email account of the user"
+                },
+                "AccountNumber": {
+                  "description": "account number of the user",
+                  "type": "number"
+                }
```

Comment:
This is too restrictive in my opinion. I'd rename it to `AccountIdentifier` and allow arbitrary strings - many identifiers out there are non-numeric.

Reply:
I'm worried that might be too much of an overlap with `AccountName`. Maybe we consolidate both into `AccountIdentifier`?

Reply:
Right, merging these makes sense to me

Reply:
Do you think we'd need `AccountEmail` still? The advantage of keeping it would making a `isThisAnEmail` check on `AccoundIdentifier` unnecessary.

Reply:
I'd say yes - emails often are used together with another identifier and both can be used for login. Therefore I'd keep this second field.